src/components/Button.js

```diff
@@ -1,7 +1,45 @@
-const Button = () => (
-  <button type="button" className="check-status">
-    Check status
-  </button>
-);
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
+
+const Button = () => {
+  const categories = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
+  const onClickStatus = (event) => {
+    dispatch(checkStatus());
+    event.currentTarget.classList.toggle('btn-off');
+
+    // API Call
+
```

Comment:
- [ ] Kindly remove all the commented code from the project, this way the project will look clean, if we want to check on the previous code, we can go to history on GitHub. This is why it's important to make a commit every time we make a change.

Reply:
Will do so, thank you!